Modules/Relationships/test/RelationshipTemplates.API.Tests.Performance.EndToEnd/src/get-relationshipTemplate-by-id.test.ts

```diff
@@ -0,0 +1,92 @@
+import http from 'k6/http';
+import { Options } from 'k6/options';
+import { describe, expect } from "https://jslib.k6.io/k6chaijs/4.3.4.2/index.js";
+
+const host = __ENV.HOST;
+const apiEndpoint = host+'/api/v1/';
+
```

Comment:
Currently there is no validation of the environment variables. This means that the error messages will probably be hard to interpret.